pos_picking_delayed/models/pos_order.py

```diff
@@ -20,23 +20,28 @@ def create_from_ui(self, orders, draft=False):
         for order_data in orders:
             session_id = order_data.get("data").get("pos_session_id")
             session = PosSession.browse(session_id)
-            order_data["data"][
-                "has_picking_delayed"
-            ] = session.config_id.picking_creation_delayed
+            order_data["data"]["has_picking_delayed"] = (
+                session.config_id.picking_creation_delayed
+                and not session.update_stock_at_closing
+            )
         return super(PosOrder, self.with_context(create_from_ui=True)).create_from_ui(
             orders, draft=draft
         )
 
-    def create_picking(self):
+    def _create_order_picking(self):
```

Comment:
Why this rename ?

Reply:
Follow up of https://github.com/odoo/odoo/commit/2f2bde5d9f647e4785ffabdc1b2e07a08722309c#diff-29cbaebb5b63b539ab173d9340b2aec87b9ad63cd322eec2347879c5412bd50b

Reply:
hum... that makes sense ! sorry for the naive question.